gcloud-java-logging/src/main/java/com/google/gcloud/logging/spi/v2/LoggingServiceV2Settings.java

```diff
@@ -312,37 +182,11 @@ public Object extractNextToken(ListLogEntriesResponse payload) {
             }
           };
 
-  private static PageDescriptor<ReadLogEntriesRequest, ReadLogEntriesResponse, LogEntry>
-      READ_LOG_ENTRIES_PAGE_STR_DESC =
-          new PageDescriptor<ReadLogEntriesRequest, ReadLogEntriesResponse, LogEntry>() {
-            @Override
-            public Object emptyToken() {
-              return "";
-            }
-
-            @Override
-            public ReadLogEntriesRequest injectToken(ReadLogEntriesRequest payload, Object token) {
-              return ReadLogEntriesRequest.newBuilder(payload)
-                  .setResumeToken((String) token)
-                  .build();
-            }
-
-            @Override
-            public Object extractNextToken(ReadLogEntriesResponse payload) {
-              return payload.getResumeToken();
-            }
-
-            @Override
-            public Iterable<LogEntry> extractResources(ReadLogEntriesResponse payload) {
-              return payload.getEntriesList();
-            }
-          };
-
-  private static PageDescriptor<
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [The field name indicates a constant but its modifiers do not](https://www.codacy.com/app/mziccard/gcloud-java/file/2388714616/issues/source?bid=3123104&fileBranchId=3228843#l185)